<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialDialog</class>
 <widget class="QDialog" name="SerialDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Serial Port</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="configGroupBox">
     <property name="title">
      <string>Configuration</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QFormLayout" name="configFormLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="portLabel">
          <property name="text">
           <string>Port</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="portComboBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="baudRateLabel">
          <property name="text">
           <string>Baud rate</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="baudRateComboBox"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="dataBitsLabel">
          <property name="text">
           <string>Data bits</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="dataBitsComboBox"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelStopBits">
          <property name="text">
           <string>Stop bits</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="stopBitsComboBox"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="parityLabel">
          <property name="text">
           <string>Parity</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="parityComboBox"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="flowControlLabel">
          <property name="text">
           <string>Flow control</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="flowControlComboBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="configButtonLayout">
        <item>
         <spacer name="configButtonSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="refreshButton">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="infoGroupBox">
     <property name="title">
      <string>Information</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="descriptionHeadingLabel">
        <property name="text">
         <string>Description:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="manufacturerHeadingLabel">
        <property name="text">
         <string>Manufacturer:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="serialNumberHeadingLabel">
        <property name="text">
         <string>Serial number:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="locationHeadingLabel">
        <property name="text">
         <string>Location:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="vendorIdHeadingLabel">
        <property name="text">
         <string>Vendor ID:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="productIdHeadingLabel">
        <property name="text">
         <string>Product ID:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="descriptionLabel">
        <property name="text">
         <string>N/A</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="manufacturerLabel">
        <property name="text">
         <string>N/A</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="serialNumberLabel">
        <property name="text">
         <string>N/A</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="locationLabel">
        <property name="text">
         <string>N/A</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="vendorIdLabel">
        <property name="text">
         <string>N/A</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="productIdLabel">
        <property name="text">
         <string>N/A</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Open</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SerialDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SerialDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
